docs/pages/machine-id/manifesto.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Machine ID Manifesto
+description: A manifesto for Machine Identity
+---
+
+As time goes on, machines and services are doing more and more impactful and
```

Comment:
Do "impactful" and "powerful" mean that services and AI agents are receiving privileges with increasing scope?

Reply:
Yes - do you think this could be clearer ? Perhaps just directly referencing scope of privileges ?

Reply:
80f34d1a73